src/Tests/Cask.Benchmarks/CompareHashedSignatureBenchmarks.cs

```diff
@@ -33,33 +33,18 @@ public void UseCompareHash()
     [Benchmark]
     public void UseHmacSha256()
     {
-        string key = Cask.Instance.GenerateKey(TestProviderSignature, "99");
-        byte[] keyBytes = Convert.FromBase64String(key.FromUrlSafe());
-        string hash = Cask.Instance.GenerateHash(TestProviderSignatureBytes, keyBytes, 32);
-        var hmac = new HMACSHA256(keyBytes);
-        byte[] hashBytes = hmac.ComputeHash(TestProviderSignatureBytes);
+        CaskKey key = Cask.GenerateKey(TestProviderSignature, "99");
+        byte[] keyBytes = Base64Url.DecodeFromChars(key.ToString().AsSpan());
+        byte[] hashBytes = new byte[HMACSHA256.HashSizeInBytes];
+        HMACSHA256.HashData(keyBytes, s_testProviderSignatureBytes, hashBytes);
+        Span<byte> computedHashBytes = stackalloc byte[32];
```

Comment:
nit to self: be consistent here with too lines above.
```suggestion
        Span<byte> computedHashBytes = stackalloc byte[HMACSHA256.HashSizeInBytes];
```